src/react-components/src/components/Link/Link.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+
+import Link from "./Link";
+import { LinkTypes } from "./LinkTypes";
+import Icon from "../Icons/Icon";
+import { iconRotationTypes } from "../Icons/IconTypes";
+
+export default {
+    title: "Link",
+    component: Link,
+};
+
+export const passedInLink = () => (
+    <Link linkType={LinkTypes.Default}>
```

Comment:
This doesn't have an underline, is that intentional?

Reply:
I haven't been able to figure this out at all!!... something is applying `text-decoration: none`, but where, I am not clear on... it's definitely not intentional

Reply:
Ah, it's not properly applying the `href` prop anymore, that's what's up